README.md

```diff
@@ -18,27 +18,31 @@
   <a href="https://pkg.go.dev/github.com/terramate-io/terramate"><img src="https://pkg.go.dev/badge/github.com/terramate-io/terramate" alt="Go Docs" /></a>
   <a href="https://goreportcard.com/report/github.com/terramate-io/terramate"><img src="https://goreportcard.com/badge/github.com/terramate-io/terramate" alt="Go Report Card" /></a>
   <a href="https://github.com/terramate-io/terramate/actions?query=branch%3Amain"><img src="https://github.com/terramate-io/terramate/actions/workflows/ci.yml/badge.svg" alt="Terramate CI Status" /></a>
-  <a href="https://github.com/terramate-io/terramate/stargazers" rel="nofollow"><img src="https://img.shields.io/github/stars/terramate-io/terramate" alt="Stars"></a>
   <a href="https://terramate.io/discord" rel="nofollow"><img src="https://img.shields.io/discord/1088753599951151154?label=Discord&logo=discord&logoColor=white" alt="Discord Server"></a>
 </p>
 
 <p align="center">
-  <a href="https://terramate.io/docs/cli">📖 Documentation</a> | <a href="https://terramate.io/docs/cli/getting-started">🚀 Getting Started</a> | <a href="https://play.terramate.io">💻 Playground</a> | <a href="https://jobs.ashbyhq.com/terramate" title="Terramate Job Board">🙌 Join Us</a>
+  <a href="https://terramate.io/docs/cli/getting-started">🚀 Getting Started</a> | <a href="https://terramate.io/docs/cli">📖 Documentation</a> |  <a href="https://play.terramate.io">💻 Playground</a>
+   <!-- | <a href="https://jobs.ashbyhq.com/terramate" title="Terramate Job Board">🙌 Join Us</a> -->
 </p>
 
 <br>
 <br>
 
 ## What is Terramate CLI?
 
-Terramate CLI is an open-source Infrastructure as Code (IaC) **orchestration tool** for Terraform, OpenTofu, Terragrunt,
-Pulumi, Cloud Formation, CDK, Azure Resource Manager (ARM), Kubernetes, and others, that **unifies**, **simplifies** and
-**scales** all your infrastructure code, tools, and workflows.  
+Terramate CLI is an open-source Infrastructure as Code (IaC) code generation and orchestration tool for Terraform,
+OpenTofu and Terragrunt. With Terramate, you can:
 
-The main value proposition of Terramate CLI is to simplify the management and improve the scalability of your
-Infrastructure as Code projects by using a modular and efficient approach called Terramate Stacks.
+(a) **Simplify complex codebases** to make your work more productive and reliable. Break down large state files with
+stacks to reduce run-time and blast radius and keep them DRY with native code generation.
 
-## Quick Install
+(b) **Automate and orchestrate Terraform, OpenTofu and Terragrunt in any CI/CD** using Pull Request automation, GitOps
```

Comment:
Should this be the first item? It seems like more of a tangible outcome than simplifying codebases, which is more of a perk.